src/libsyntax/feature_gate.rs

```diff
@@ -689,6 +687,8 @@ declare_features! (
     (accepted, repr_packed, "1.33.0", Some(33158), None),
     // Allows calling `const unsafe fn` inside `unsafe` blocks in `const fn` functions.
     (accepted, min_const_unsafe_fn, "1.33.0", Some(55607), None),
+    // support for arbitrary delimited token streams in non-macro attributes
+    (accepted, unrestricted_attribute_tokens, "1.33.0", Some(55208), None),
```

Comment:
```suggestion
    (accepted, unrestricted_attribute_tokens, "1.34.0", Some(55208), None),
```

The master=>beta promotion happens on the 15th of January and FCP takes at least 10 days so this won't make it to 1.33. So this will be 1.34 at earliest unless we beta-backport (which we usually don't do for new features...?)